src/services/TestDataService.ts

```diff
@@ -633,6 +634,27 @@ export class TestDataService {
         return rule;
     }
 
+    /**
+     * Creates a new basic landing page layout.
+     *
+     * @param cmsPageType - The type of the cms page layout (page/landingpage/product_detail...).
+     * @param overrides - Specific data overrides that will be applied to the cms page layout data struct.
+     */
+    async createBasicLandingPageLayout(cmsPageType: string, overrides: Partial<CmsPage> = {}): Promise<CmsPage> {
```

Comment:
The user can add a pageType here therefore I would recommend to call the method:

`createBasicPageLayout`